runtime/jcl/common/mgmtmemory.c

```diff
@@ -702,6 +703,31 @@ Java_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificati
 				return;
 			}
 
+			/* error tracepoint for the cases
+		    the value of init or max is negative but not -1; or
+		    the value of used or committed is negative; or
+		    used is greater than the value of committed; or
+		    committed is greater than the value of max if msx is not -1.
+		    */
```

Comment:
Please indent only with tabs (except for the space before `*` for continued lines); see [1]. Sentences should begin with a capital, and it would probably help to list the cases on separate lines.

[1] https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-comments